hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -36,354 +32,251 @@ import java.time.LocalDateTime
  * look in JSON format see test class [AttributeSearchRequestJsonTest].
  */
 class AttributeSearchIntegrationTest : AbstractSearchIntegrationTest() {
-  private val prisonerBuilders = mutableListOf<PrisonerBuilder>()
-
-  // The super class refreshes the index before this test class runs, but we don't want it to load any prisoners yet
-  override fun loadPrisonerData() {}
-
-  @BeforeEach
-  fun `clear search repository`() {
-    prisonerBuilders.onEach { prisonerRepository.delete(it.prisonerNumber) }.clear()
-    waitForPrisonerLoading(expectedCount = 0)
+  // Create 3 prisoners with predictable data we can search for
+  private final val prisonerCount = 3
+  val prisoners =
+    (1..prisonerCount).map {
+      PrisonerBuilder(
+        prisonerNumber = "P$it",
+        firstName = "John$it",
+        dateOfBirth = LocalDate.parse("1989-12-31").plusDays(it.toLong()).toString(),
+        heightCentimetres = 180 + it,
+        recall = it % 2 == 0,
+        currentIncentive = IncentiveLevelBuilder(levelDescription = "I$it", dateTime = LocalDateTime.now().minusDays((prisonerCount - it).toLong())),
+        alertCodes = listOf("AT$it" to "AC$it"),
+        profileInformation = ProfileInformationBuilder(youthOffender = it % 2 == 1),
+      )
+    }
```

Comment:
I dislike this, but once you understand what it does you can understand all the tests. Might be better to explicitly. Might be better to just add another 20 lines and create each prisoner builder explicitly.